tests/src/python/test_qgsexpression.py

```diff
@@ -79,6 +77,30 @@ def raise_exception(feature, parent):
         # an undefined variable
         foo  # noqa: F821
 
+    @qgsfunction(group='testing', register=False)
+    def simple_sum(val1, val2):
+        return val1 + val2
+
+    @qgsfunction(group='testing', register=False)
+    def sum_vargs(val1, *args):
+        return val1 + sum(args)
+
+    @qgsfunction(group='testing', args=-1, register=False)
+    def func_params_as_list_legacy(values, feature, parent):
+        return values
+
+    @qgsfunction(group='testing', params_as_list=True, register=False)
+    def func_params_as_list(values):
+        return values
+
+    @qgsfunction(group='testing', register=False)
+    def func_params_no_list(values):
+        return values
+
+    @qgsfunction(group='testing', register=False)
+    def func_feature(*args, feature):
+        return (feature["a"] + sum(args)) * feature["b"]
```

Comment:
Is there already a test function with `context`?

Reply:
I added one